manifests/alertmanager.pp

```diff
@@ -59,6 +59,8 @@
 #  If package, then use this for package ensure default 'latest'
 # @param package_name
 #  The binary package name - not available yet
+# @param config_dir
```

Comment:
can you please put the not systemd_exporter changes into a dedicated PR?

Reply:
I can do that !